include/revng/PipeboxCommon/CliftContainer.h

```diff
@@ -73,9 +77,21 @@ class CliftFunctionContainer {
     return Result;
   }
 
+  void setIsExpiring() { Expiring = true; }
+
+  void pruneIfPossible() {
+    if (not Expiring)
+      return;
```

Comment:
Try to be consistent `if (Expering) { ... }` vs `if (not Expiring) return`.

Reply:
Stuck with `if (not Expiring) return;`